nav2_costmap_2d/plugins/static_layer.cpp

```diff
@@ -132,6 +132,7 @@ StaticLayer::getParameters()
   declareParameter("map_subscribe_transient_local", rclcpp::ParameterValue(true));
   declareParameter("transform_tolerance", rclcpp::ParameterValue(0.0));
   declareParameter("map_topic", rclcpp::ParameterValue(""));
+  declareParameter("footprint_clearing_enabled", rclcpp::ParameterValue(true));
```

Comment:
```suggestion
  declareParameter("footprint_clearing_enabled", rclcpp::ParameterValue(false));
```

Lets not change default behavior - but LGTM with documentation update to include param + migration guide